packages/react-components/src/templates/__tests__/ManuscriptForm.test.tsx

```diff
@@ -32,24 +33,73 @@ it('renders the form', async () => {
   expect(screen.getByRole('button', { name: /Submit/i })).toBeVisible();
 });
 
-it('title is sent on form submission', async () => {
+it('data is sent on form submission', async () => {
   const onSave = jest.fn();
   render(
     <StaticRouter>
-      <ManuscriptForm {...defaultProps} onSave={onSave} />
+      <ManuscriptForm
+        {...defaultProps}
+        title="manuscript title"
+        type="Original Research"
+        lifecycle="Draft manuscript"
+        onSave={onSave}
+      />
     </StaticRouter>,
   );
 
-  userEvent.type(
-    screen.getByRole('textbox', { name: /Title of Manuscript/i }),
-    'manuscript title',
-  );
   userEvent.click(screen.getByRole('button', { name: /Submit/i }));
   await waitFor(() => {
-    expect(onSave).toHaveBeenCalledWith({ title: 'manuscript title', teamId });
+    expect(onSave).toHaveBeenCalledWith({
+      title: 'manuscript title',
+      versions: [{ type: 'Original Research', lifecycle: 'Draft manuscript' }],
+      teamId,
+    });
   });
 });
 
+it('does not display the lifecycle select box until type is selected', async () => {
+  render(
+    <StaticRouter>
+      <ManuscriptForm {...defaultProps} />
+    </StaticRouter>,
+  );
+  expect(
+    screen.queryByLabelText(/Where is the manuscript in the life cycle/i),
+  ).not.toBeInTheDocument();
+
+  const textbox = screen.getByRole('textbox', { name: /Type of Manuscript/i });
+  userEvent.type(textbox, 'Original');
+  userEvent.type(textbox, specialChars.enter);
+  textbox.blur();
+
+  expect(
+    screen.getByRole('textbox', {
+      name: /Where is the manuscript in the life cycle/i,
+    }),
+  ).toBeInTheDocument();
+});
+
+const manuscriptTypeLifecyclesFlat = manuscriptTypeLifecycles.flatMap(
+  ({ types, lifecycle }) => types.map((type) => ({ type, lifecycle })),
+);
+it.each(manuscriptTypeLifecyclesFlat)(
+  'displays $1 lifecycle option for when $2 type is selected',
```

Comment:
I think we can use `$lifecycle` and `$type` because manuscriptTypeLifecyclesFlat gives an array with objects like 

```
{ type: 'Original Research', lifecycle: 'Preprint, version 1' }
```


```suggestion
  'displays $lifecycle lifecycle option for when $type type is selected',
```